src/base/projects/Commit/CommitVerifiedBadge.svelte

```diff
@@ -1,14 +1,25 @@
 <script lang="ts">
   import type { CommitMetadata } from "@app/commit";
-  import CommitAuthorship from "./CommitAuthorship.svelte";
+
+  import { debounce } from "lodash";
+
```

Comment:
I like using lodash to avoid implementing it ourselves 😃 but we should definitly only load `/debounce` since `lodash` is huge.
Or maybe even `lodash.debounce`

```suggestion
  import { debounce } from "lodash/debounce";
```

Reply:
Thing is, `loadash` has always been in our dependency tree. Just not explicitly at the top. Grep the lockfile for it and you'll see ; P

Reply:
There's also other goodies in there that I'd love to start using in other places, e.g. `throttle`, `orderBy`, `isEqual`, etc.

Reply:
When would we see a performance hit though? Because the bundler should be smart enough to tree-shake anything unneeded, no?

b3b3c06a03c05e1fb53cca93fbf460ee314b5e1c

Reply:
Here is a good article about this, seems like it still is better if we import it with the dash
https://blog.battlefy.com/tree-shaking-lodash-with-vite

Reply:
Ugh! Good to know! 🙇